packages/web/env-config.js.template

```diff
@@ -11,6 +11,9 @@ const environments = {
     BLOCKSCOUT: {
       pageLength: 5,
       uri: 'http://localhost:4000/graphiql',
+      mainnet: 'http://localhost:4000/graphiql',
+      baklava: 'http://localhost:4000/graphiql',
```

Comment:
opinion: I know they were here before, but im not sure these URLs need to be in the ENV config file. only real purpose would be to either obscure them or to have different environments point to different networks but i think that is just a recipe for not noticing a network isnt working until production.

Reply:
How do you think that we should handle that?

Reply:
the network => url object could be kept either in the ValidatorList component or its own file.

unless you see a benefit of having them inside the ENV file? 

up to you.

Reply:
It's true that probably we won't need different variables by environment. So ok, I'll move to the component